Python/Include/Constant.hpp

```diff
@@ -23,6 +23,10 @@ namespace Rover {
     pybind11::class_<Constant<T>>(module, name.c_str())
       .def(pybind11::init<T>())
       .def("generate", &Constant<T>::generate);
+    implicitly_convertible<Constant<T>, Box<pybind11::object>>();
```

Comment:
Qualify the function call, ie: ```pybind11::implicitly_convertible```

Reply:
Include ```implicitly_convertible<Constant<T>, Box<T>>```.

Reply:
Works but one has to export Box<T> first, otherwise will get a runtime error while importing. The same applies to Box<object> and Constant<object> in fact.